runhouse/resources/hardware/cluster.py

```diff
@@ -1645,3 +1658,30 @@ def _check_for_child_configs(cls, config: dict):
             config["default_env"] = default_env
 
         return config
+
+    def _disable_status_check(self):
+        """
+        Stopping consecutively sending status to Den.
+        """
+        if not self.den_auth:
```

Comment:
what happens if i have a cluster with den auth enabled, then i decide to disable it. Will the status check still continue in this case?